transport/grpc/outbound.go

```diff
@@ -170,7 +171,7 @@ func (o *Outbound) invoke(
 		return err
 	}
 
-	return transport.UpdateSpanWithErr(
+	invokeErr := transport.UpdateSpanWithErr(
```

Comment:
nit: calling this invokeErr not necessary. Use the standard variable name: `err`.

(It was called `invokeErr` in the other case because we didn't want it to be shadowed by the `err` from `getApplicationHeaders`.)

Reply:
resolved, thanks for explanation.